docs/sources/patterns/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+aliases:
+  - /docs/explore-apps/logs/patterns
+description: 
+keywords:
+  - Logs
+  - Log Patterns
+  - Explore
+  - Patterns
+  - Drain
+  - Categorization
+  - Analysis
+menuTitle: Log patterns
+title: Log patterns
+weight: 800
+---
+
+# Log patterns
+
+Log patterns allow you to work with groups of similar log lines. They are automatically extracted when your logs are ingested.
+
+The Explore Logs app shows you the patterns alongside their log volumes. From this view, you can investigate spikes and drill down to view the logs.
+
+## More use cases
+
+Log patterns allow you to:
+
+* Browse the log volume over time of different types of logs.
+* Simplify log management by grouping similar log entries.
+* Enhance log searches by focusing on relevant patterns.
+* Improve troubleshooting efficiency by highlighting critical log lines.
+* Reduce storage requirements by minimizing log data.
+* Filter out noisy log lines during exploration.
+* Identify specific log lines for targeted analysis.
+
+## Quick guides
+
+We've outlined the steps you'll need to take to perform these common use cases.
+
+### Browse log volumes by type
+
+
+
+### Targetted analysis
+
+If you know the kind of log line you're looking for, Log patterns are an easy way to get everything else out of the way.
+
+To view only a specific set of patterns, perform the following steps:
+
+1. Go to **Explore > Logs**
+2. Select the relevant **Service**
+3. Open the **Patterns** tab
+4. Identify a pattern that matches the type of logs you're interested in
+5. Select **Include**
+6. Go to the **Logs** tab and notice the filtered view
+
+### Hide noise
```

Comment:
Maybe something more generic like "Hide unwanted log lines"

Reply:
I think the terminology 'noise' matters here - went with "Hide noisy log lines". But this raises an interesting question about the level of knowledge we want to assume; walk the line between clarity and patronisation :)